mdm/internal/checkinproto/checkin.pb.go

```diff
@@ -1,11 +1,15 @@
-// Code generated by protoc-gen-go. DO NOT EDIT.
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
This doesn't seem right. Seems like `protoc-gen-gogo` generated this file but this file should be generated with normal `protoc-gen-go`. How did you create this file? `go generate` should be all that's needed once the required `protoc` and `protoc-gen-go` is installed.

Reply:
I had some trouble to run just _go generate_ . I don't recall what I did exactly.

I tried to do everything by the book in the later commit. Still couldn't get the go generate to exit without errors. After some googling found a hint to add extra option to the proto file. After doing that, go generate exited ok.

I noticed there was some discussion in #757 about same kind of difficulties.